src/train_model.py

```diff
@@ -640,7 +696,8 @@ def csordas_loss(net_out, data):
 
 
 @train_exp.automain
-def run_training(dataset, net_type, net_choice, optim_func, optim_kwargs):
+def run_training(dataset, net_type, net_choice, optim_func, optim_kwargs,
+                 simple_net_kwargs):
```

Comment:
please add line in docstring about what simple_net_kwargs is

Reply:
also, that should probably just be a net_kwargs variable that gets fed into every net construction (and is just empty for most networks).